contracts/admin/src/common/type_defs.rs

```diff
@@ -0,0 +1,12 @@
+use super::{CodeIdWithMigrateMsg, GeneralContracts, LpnContracts};
+
+pub type MigrateContract = CodeIdWithMigrateMsg<String>;
+pub type MaybeMigrateContract = Option<MigrateContract>;
+
+pub type MigrateGeneralContract = CodeIdWithMigrateMsg<String>;
+pub type MaybeMigrateGeneralContract = Option<MigrateGeneralContract>;
+pub type MigrateGeneralContracts = GeneralContracts<MaybeMigrateGeneralContract>;
```

Comment:
on the other hand side, the types `GeneralContracts<T>` and `LpnContracts<T>` with `T=Addr` are left not defined.